### ISSUE TITLE
Feedbak from Sonia

### ISSUE THREAD

**Issue by soniaguera** [s-andrews/virusbreak#43]
Hello Simon and Adrian, 

The game is amazing! It is a fab tool. Really fun to see how we can play with the setting and the difference between the virus. Really cool! Kids are going to love the infect button.

As a general feedback, I would cut the text.It works better if the user finds how to do things rather than just being told how to do them ("learn by doing pedagogy" ). I would modify things in that direction (some thoughts below). 

Thanks!!

------------------

1- The first page might be too heavy on text. I would try to add more graphics and reduce text as follows:

                  A/ one page with a big banner (Wellcome to Virus Break and virus icons) and the game inttro text. 
                 B/ Remove the "How to play" text from there. Include it as a tab betlow the title inside the game (above virus, properties and run).
                 C/Shorten paragraphs in the how to play text (I could include suggestion in a separate issue thread.

2- I would guide the user to run the preset properties first. Otherwise we move the parameters first and it is confusing. I think if we move the RUN tab to the middle will be better. 

3- Add a reset bottom in the parameters section. We move them too quickly and then we dont know the original :( Add a line saying: these are the properties of the virus, you can change them in the slide bar and reset them here).

4- The page will benefit from including some evidence on the assumptions of the simulation. e.g. Refs for the viral parameters, economic cost and NHS costs calculations. This could be lab page (below the game). The model is only for England?

5- A general question: why each square is 11k carriers? Not sure how this number was chosen.

6- Could we add a Quiz tab below the simulations? Add the following questions: 1 - Why measles spread slowly?  2- How you could reduce the fast spread of coronavirus? 3- How many ebola carriers need to start the infection to achive similar spread speed to coronavirus? Send us your answers by twitter to XXX 

7- Rather than coronavirus, should it be Coronavirus (SARS-Cov-2)? Most of the coronavirus would have very different parameters.

8- I would like to include a little activity for teachers, but I need to think about it a bit more. I will create a separate issue with it. I can make a .doc for teachers to use along the game online. Teachers and children love games!

----------------------

Not sure how easy is to change these things, let me know your thoughts on them and if I could help somehow (although I can not code :)

**Comment by s-andrews** [comment]
1. Those all sound good.  I’d be very happy to get suggestions for updated text for the instructions.  It should be easy enough to have a little help icon somewhere which brings up a help screen if you need it. 

2. Yes, that makes sense.  I can put the presets up first.  Having the run tab in the middle is only an issue on a phone (you can have them all up at the same time on a desktop).  I think it’s easy enough to move the run to the middle?

3. I need to check on this one – I need to check if we save the state for which preset  was last selected to be able to reset to.  Should be a way around it if not.

4. This should probably go in a revamped help page.  The population is the UK (66M people)

5. It’s the total population of the UK divided by the number of squares.  We can round it down to 10k if you like.  There’s probably more than a 10% margin of error in most of the numbers anyway?

6. I think having some guided questions to go along with the game will really help it.  I’m not really sure where the best place to put these would be.  We could have a random question which pops up after each simulation, we could put things into the help or we could have a separate page to the simulation with more space to expand on it.  As with some of the other recent suggestions on of the big problems now is that if we want the whole thing to fit on one screen then we’re basically full now – on many phones there’s simply no more space in the existing interface, so any new thing which gets added need something to be removed.

7. That would be equally true for influenza too.  Putting that much extra text will mess up the layout.  That kind of detail feels like it would be better in the help where we say how the numbers are justified.

**Comment by s-andrews** [comment]
Other than suggestions the best help would be if you could rewrite the help or quiz instructions.  It doesn't really matter what format that comes in - plain text is fine - that's then easy for me to insert into the code.  It's the content which is most important.

**Comment by soniaguera** [comment]
Thank you Simon! Some thoughts:

-----

1- Content/ Im uploading the content for Help and Quiz in a separate issue. 

2- Run tab/ somewhere that we find it earlier than the virus properties will be good. Wherever it is easier for you to code it and it works on a phone. 

 3- Reset properties/ If coding the reset button is difficult, we can just include them as text next to each slide bar or somewhere else. Whatever is easier.  

 4/ Population and squares/ I will add in the lab page rather than in the game as Adrian suggested. (that the model is defined for UK (66M people) and 11k carriers (each square)). 

5/ Quiz location/ If you find a good place for the Quiz, we can have it in the game, but there is no need  the extra work to remove things around then. I will can add the QUiz into the lab website if needed. Just let me know.

7/ Names of virus/  Game stays as it is, and I specify in the website that we use SARS-covid-2 and the others are: H1N1, Paramyxoviridae and Ebolavirus. Is this correct? 

-----
To sum up modifications:
1- Help and Quiz modifications. They can go into labwebsite if difficult. 
2- Reset properties and Run button/ If reset button is hard to code, we just add the text, and it would be nice if the RUN button could be placed in a way that we find it before than the properties. 
3- The rest goes into the lab page

----

Cheers!
One extra question: what is the switch button for? It seems to me that nothing happens...
